src/Microsoft.AspNetCore.Razor.Tools/project.json

```diff
@@ -1,11 +1,16 @@
 {
   "version": "1.0.0-*",
-  "description": "TagHelper tooling for .NET Core CLI.",
+  "description": "TagHelper tooling for .NET Core CLI. Contains the dotnet-razor-tooling command used to resolve TagHelperDescriptors for projects at design-time.",
   "packOptions": {
     "repository": {
       "type": "git",
       "url": "git://github.com/AspNet/RazorTooling"
-    }
+    },
+    "tags": [
+      "taghelpertooling",
```

Comment:
Nix this tag, it's pretty much just repeating the package id. Plus, this tag would clearly never be used on any other package 😄 

Some tags to be inspired by: (i.e. copy the ones that make sense)
* https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor/project.json#L17
* https://github.com/aspnet/Razor/blob/dev/src/Microsoft.AspNetCore.Razor.Runtime/project.json#L18
* https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.TagHelpers/project.json#L9
* https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Razor/project.json#L9